test/benchmarks/mongoBench/runner.js

```diff
@@ -57,6 +57,7 @@ class Runner {
     this.minExecutionTime = options.minExecutionTime || CONSTANTS.DEFAULT_MIN_EXECUTION_TIME;
```

Comment:
```js
const sortedData = [].concat(rawData).sort();
```

I cannot comment on the line directly. This does not sort the values correctly

```js
[].concat([1.2, 10.1, 2.4, 20.2]).sort();
[ 1.2, 10.1, 2.4, 20.2 ]
```
10 doesn't come before 2 😢

Reply:
It does in the alphabet 😿